### PR TITLE
Create install.sh

### PR DESCRIPTION
**Pull request by hackertron**
install pngtools on debian/ubuntu based Linux distro's !!

pngtools : pngtools: series of tools for PNG (Portable Network Graphics) images

Usage : For various analysis of related to PNG's

### CODE REVIEW

**Comment by zardus** [comment]
Could you add pngtools to the readme?

**Comment by hackertron** [comment]
I have added pngtools in readme  now.

**Comment by zardus** [comment]
Ah, I just looked at the actual install script (was on mobile before and just looked at the list of files). A few things:

1. ctf-tools is meant to include tools that are either not in the official repositories or that are outdated in the official repositories. I'd like to avoid a bunch of install scripts that just apt-get stuff. Instead, there is a section in the readme (https://github.com/zardus/ctf-tools#tools-in-the-official-debianubuntu-repos) where these things should be added (instead of being actual install scripts that just call apt-get).
2. It's not relevant for this PR (as we don't need an install file for it, just a readme entry in the right section), but the install file should be called `install`, not `install.sh`.
3. By decree of myself, `install` should perform no actions that require sudo privileges. That's what `install-root` is for. There's few things I hate more than random install scripts from the internet asking for my password :-). Again, this isn't relevant for this PR, as pngtools can just be an entry in the right spot.

In summary, for this PR, rather than having a script that just apt-get installs the tool, we should just have it listed in the appropriate section of the readme.

**Comment by hackertron** [comment]
Okay i understood now how all this should work !!

* removed pngtools folder and that install.sh script

I have just added pngtools in readme with launchpad link !! Is it fine now ??

**Comment by zardus** [comment]
Almost! It should be in a different section, lower in the README: https://github.com/zardus/ctf-tools#tools-in-the-official-debianubuntu-repos

**Comment by hackertron** [comment]
Okay fixed it  👍

**Comment by zardus** [comment]
Sweet! Almost there :-)

Could you change the tag on it from `<!--tool--><!--test-->` to `<!--deb-tool-->`, like the others in the section? Otherwise, the CI will try to test it.

I think after that, we should be all set!

**Comment by hackertron** [comment]
Changed it to  !!

**Comment by zardus** [comment]
Awesome stuff! Squashed and merged :-)
